### PR TITLE
🌫 Add a Notifications panel

### PR DESCRIPTION
**Pull request by p-sad**
See #851 
Please understand that the mere awareness of CSS existing makes me go to sleep with the lights on

### CODE REVIEW

**Comment by p-sad** [comment]
This PR is for just the component, logic coming next

File: packages/ui/src/common/components/Notifications/Notifications.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import { SidePaneHeader, SidePanelTop } from '../SidePane'
+import { Label } from '../typography'
+
+import { NotificationsBody, NotificationsPanel } from './NotificationsPanel'
+
+interface Props {
+  onClose: () => void
+}
+
+export const Notifications = ({ onClose }: Props) => {
```
Comment (jodator) [inline packages/ui/src/common/components/Notifications/Notifications.tsx:15]: A storybook story is needed for new component

File: packages/ui/src/common/components/Notifications/NotificationsButton.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, { useRef, useState } from 'react'
+
+import { useOutsideClick } from '../../hooks/useOutsideClick'
+import { ButtonPrimary } from '../buttons'
+import { HelpIcon } from '../icons'
```
Comment (jodator) [inline packages/ui/src/common/components/Notifications/NotificationsButton.tsx:5]: There must a bell icon :pray: this one is ugly :)

**Review by jodator — CHANGES_REQUESTED** [review]
A story book is needed

**Pull request merged**